### ISSUE TITLE
App freezes at 1:50 two times then works 

### ISSUE THREAD

**Issue by DrJoe61** [IanPeake/FatMaxxer#42]
![Screenshot_20220825-162239_FatMaxxer](https://user-images.githubusercontent.com/112136080/186788837-ae6e7a3e-6e3b-4785-abe2-72932ad26cf5.jpg)
**Describe the bug**

**FatMaxxer version number or name**
- The version name can be found under App Info while the app is running

**Phone Manufacturer, Model and Android Version**
- Manufacturer
- Model

**Comment by DrJoe61** [comment]
![Screenshot_20220825-175600_Google Play Store](https://user-images.githubusercontent.com/112136080/186788929-0a49fdd4-4115-40ff-8f48-ad04e44d6c16.jpg)

**Comment by DrJoe61** [comment]
Love what this app does...  better than hrv4training

**Comment by MedTechCD** [comment]
> Love what this app does... better than hrv4training

You absolutely don't know what you're talking about. HRV4Training is an app to measure your resting HRV. Fatmaxxer measures HRV during exercise. Both apps do a fantastic job in their respective domain and have absolutely nothing to compare. They are complementary.

**Comment by DrJoe61** [comment]
YEAH . . . I realized the moment I sent it that I was referring to HRV
Logger...   as you well know..

What about your app freezing at 1:50 for me ??

and...    do you have some feedback you can give me as to

What should my exercise heart rate do as I become more aerobically fit ??

I mean...   with training. over time.  my heart rate should be .. HIGHER at
the point I reach 0.75 zone ?

J
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -

Dr Joe DiDuro  DC MS DABCN

**Comment by MedTechCD** [comment]
I'm not the dev, I'm a user...
Sideload app-debug.apk from the code page. It is a more recent version. I do occasionally have the same problem but it's minor.
Read up on what you're trying to do. HR should go down for a1=0.75 because that means your Aerobic threshold goes up.
Check muscleoxygentraining.com for plenty of information.
And Hold your horses before complaining about other apps if you don't know what you're doing.

**Comment by DrJoe61** [comment]
Thanks for your time and attention to this matter..

On Sat, Sep 10, 2022, 3:11 PM MedTechCD ***@***.***> wrote:

> I'm not the dev, I'm a user...
> Sideload app-debug.apk from the code page. It is a more recent version. I
> do occasionally have the same problem but it's minor.
> Read up on what you're trying to do.
>

HR should go down for a1=0.75 because that means your Aerobic threshold
> goes up.
> Check muscleoxygentraining.com for plenty of information.
>

I have read all of Dr Roger's work..  I don't recall him explaining this
point.. Please correct me with the relevant reference.

I hear what you are saying...   let me explain it better..

Your conclusion may be worded differently..

Resting HR should go down as you become more fit.

The Working HR ... the HR training @ a1.. might in fact be higher as you
become more fit.

As your aerobic threshold improves..  you should be able to produce greater
output power with the same or lower Active HR..

Or arrive at a HIGHER Active HR before you reach LT1.

And Hold your horses before complaining about other apps if you don't know
> what you're doing.
>

I'm not complaining...  what part do I not know what I'm doing?

**Comment by ensingerphilipp** [comment]
Seeing the same problem - even built the app myself but it freezes at 1:50
